CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ Version 2.1.0 of the App Center React Native SDK includes a new module: Auth.
 
 ### iOS
 
-* **[Fix]** Fix calling UI API on background thread.
+* **[Fix]** Fix a crash that could occur when delaying the start of the Push module.
```

Comment:
The bug reported in https://github.com/microsoft/appcenter-sdk-react-native/issues/598, I don't see "crash" is mentioned anywhere in the Issue. 

I am curious if this bug would actually cause the app to crash. 

Maybe I am being picky here, if this bug doesn't crash the app, we can rephrase the wording to something like this:
```
Fix a bug where SDK calls UI API on a background thread when delaying the start of the Push module.
```

Reply:
Ok, but now I think the bug should have been fixed on native side.